ibis/backends/sql/compilers/duckdb.py

```diff
@@ -349,7 +349,14 @@ def visit_UnwrapJSONBoolean(self, op, *, arg):
         )
 
     def visit_ArrayConcat(self, op, *, arg):
+        # TODO: duckdb 1.1.3 was the first release to support list_concat(a,b,c,d,...)
```

Comment:
We can't do this because list_concat will always follow the postgres array_cat behavior, which is NOT to propagate NULLs.

Reply:
Oops, yes that's correct. I will fix these up